ui/app/js/controllers/permission_scheduled_changes_controller.js

```diff
@@ -112,6 +134,72 @@ function($scope, $routeParams, $location, $timeout, Permissions, Rules, Search,
     });
   };
 
+  $scope.signoff = function(sc) {
+    var modalInstance = $modal.open({
+      templateUrl: "signoff_modal.html",
+      controller: "SignoffCtrl",
+      backdrop: "static",
+      resolve: {
```

Comment:
It looks like this isn't really practical, since these (particurarly in the required signoffs controller) depend on args being passed in from the templates. It's possible to factor some of this out, but the added indirection makes it harder to read IMO.